src/app/common/app-nav/app-nav.component.js

```diff
@@ -0,0 +1,9 @@
+import templateUrl from './app-nav.html';
```

Comment:
this should be `template` instead of `templateUrl` since import import the html content of the file